shared/chat/conversation/list.desktop.js

```diff
@@ -74,14 +90,24 @@ class ConversationList extends Component<void, Props, State> {
   componentWillReceiveProps (nextProps: Props) {
     // If we're not scrolling let's update our internal messages
     if (!this.state.isScrolling) {
+      this._invalidateChangedMessages()
```

Comment:
Huh, now I'm confused about how this works too.  `this.props` has been updated with the new messages but `this.state` hasn't yet.  @chrisnojima, do you remember why this works?